templates/manifests.yaml

```diff
@@ -0,0 +1,83 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Amazon S3 Find and Forget Manifests stack
+
+Resources:
+  ManifestsBucket:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      VersioningConfiguration:
+        Status: Enabled
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+      LifecycleConfiguration:
+        Rules:
+          - Id: ExpireDeletedManifests
+            Prefix: manifests/
+            Status: Enabled
+            NoncurrentVersionExpirationInDays: 1
```

Comment:
question: would there be any value in retaining the manifest beyond 1 day e.g. the same job data retention setting. I suspect not but worth asking!

Reply:
Not sure I understand the question. Currently, the manifests are deleted (and their partitions) by the stream processor when the DDB job document expires, so I think it has the same data retention as the job data currently. This is only for deleting the thing indefinitely as the bucket has versioning enabled - so the deletion actually just creates a deletion marker. I could potentially change the stream to fetch the version first and do a hard deletion but perhaps that would be a bit more expensive (computationally + S3 calls) so that's why I decided to do it like this.

Reply:
doh, I missed the noncurrent bit